packages/pulpcore/python-setuptools-scm/python-setuptools-scm.spec

```diff
@@ -26,6 +26,7 @@ use_scm_version...
 Summary:        %{summary}
 %{?python_provide:%python_provide python3-%{pypi_name}}
 
+Requires:       python3-toml
```

Comment:
Nit: perhaps this should have an empty line after it

Reply:
It's a "bug" in our jinja2 template, but yeah I can move that manually.